exporter/ethstore.go

```diff
@@ -48,11 +48,22 @@ func StartEthStoreExporter(bnAddress string, enAddress string, updateInterval, e
 		ese.Sleep = time.Minute
 	}
 
+	// Reexport days if specified
+	if startDayReexport != -1 && endDayReexport != -1 {
+		for day := startDayReexport; day <= endDayReexport; day++ {
+			err := ese.exportDay(strconv.FormatInt(day, 10), true)
+			if err != nil {
+				logger.WithError(err).Errorf("error reexporting eth.store day %d in database", day)
+				return
+			}
+		}
+	}
+
 	ese.Run()
 
 }
 
-func (ese *EthStoreExporter) ExportDay(day string) error {
+func (ese *EthStoreExporter) exportDay(day string, shouldClearDay bool) error {
```

Comment:
**Beginner's question:** Why can you use a lower case letter here? This means that the function is not exported anymore, why does https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4ad8da36a4a34dfa029705fc50f96a971f7dd85e/exporter/ethstore.go#L54 still work?

Reply:
Yes, the function is no longer exported so it can only be used within the package exporter.
We are still in that package.

Reply:
But this function is part of a struct so I expected it to act like a private method in C++: It cannot be called from outside the struct, even in the same package.
Looks like Go works differently here though. Unfortunately that means that a function in a struct cannot be "hidden" completely.